edb/schema/delta.py

```diff
@@ -721,10 +736,12 @@ def __init__(
         op: Command_T,
         *,
         modaliases: Optional[Mapping[Optional[str], str]] = None,
+        ignorenames: Set[str] = frozenset(),
```

Comment:
`ignorenames` is too generic of a name to be used for a very specific purpose in the general context.  How about `localnames`?  Please also add the type annotation above and a comment on it with the purpose of the attribute.

Reply:
Sure, `localnames` sounds good.